pkgdb2client/utils.py

```diff
@@ -187,6 +187,27 @@ def __get_fas_user_by_email(email_address):
     return user
 
 
+def get_fasinfo(email):
+    ''' Get fas username and build a name string for the user like:
+        Human Name (fas_username) <email>
+    '''
+
+    fas_user = __get_fas_user_by_email(email)
+
+    fas_username = fas_user["username"]
```

Comment:
On Mon, Feb 08, 2016 at 11:38:03PM -0800, Pierre-Yves Chibon wrote:
> > @@ -187,6 +187,27 @@ def __get_fas_user_by_email(email_address):

> > +    fas_user = __get_fas_user_by_email(email)
> > +
> > +    fas_username = fas_user["username"]
> 
> This won't work if/when ``fas_user`` is None.

Thank you, I addressed this now as well.